components/block-toolbar/index.js

```diff
@@ -7,7 +7,7 @@ import classNames from 'classnames';
  * Internal dependencies
  */
 import './style.scss';
-import Dashicon from 'components/dashicon';
+import Dashicon from 'dashicon';
 
 function Toolbar( { controls } ) {
```

Comment:
Here and elsewhere, if we're changing the filename the component name should also change (to `BlockToolbar` in this case).